--- src/netlib.c-orig	2007-10-18 00:16:02.000000000 +0300
+++ src/netlib.c	2008-12-15 08:09:59.000000000 +0200
     }
@@ -1990,8 +2080,13 @@
 
 #if defined(__CPU_SETSIZE)
 #define NETPERF_CPU_SETSIZE __CPU_SETSIZE
+#if defined(__CPU_SET_S)
+#define NETPERF_CPU_SET(cpu, cpusetp)  __CPU_SET_S(cpu, sizeof (cpu_set_t), cpusetp)
+#define NETPERF_CPU_ZERO(cpusetp)      __CPU_ZERO_S (sizeof (cpu_set_t), cpusetp) 
+#else
 #define NETPERF_CPU_SET(cpu, cpusetp)  __CPU_SET(cpu, cpusetp)
 #define NETPERF_CPU_ZERO(cpusetp)      __CPU_ZERO (cpusetp)
+#endif
   typedef cpu_set_t netperf_cpu_set_t;
 #else
 #define NETPERF_CPU_SETSIZE sizeof(unsigned long)
