diff -Nru slmodem-2.9.11-20080417-orig/drivers/Makefile slmodem-2.9.11-20080417/drivers/Makefile
--- slmodem-2.9.11-20080417-orig/drivers/Makefile	2008-04-26 17:00:32.565339319 +0300
+++ slmodem-2.9.11-20080417/drivers/Makefile	2008-04-26 17:02:54.367339043 +0300
@@ -37,7 +37,6 @@
 all install uninstall: kernel-ver
 	$(MAKE) $@ KERNEL_VER=$(shell ./kernel-ver)
 
-install: install-devices
 uninstall: remove-devices
 
 kernel-ver::
@@ -84,7 +83,6 @@
 	cp /etc/modules.conf /etc/modules.conf.slamr && \
 	    echo 'alias char-major-242 slamr' >> /etc/modules.conf && \
 	    echo 'alias char-major-243 slusb' >> /etc/modules.conf 
-	/sbin/depmod -a
 uninstall:
 	/sbin/modprobe -r slamr slusb
 	cp /etc/modules.conf /etc/modules.conf.slamr && \
@@ -113,7 +111,6 @@
 	install -D -m 644 slamr.ko $(module-dir)/slamr.ko
 endif
 	install -D -m 644 slusb.ko $(module-dir)/slusb.ko
-	/sbin/depmod -a
 uninstall:
 ifeq ($(ARCH64), 0)
 	modprobe -r slamr ; echo -n
